### PR TITLE
effect.dm cleanup and cosmetic babel syndrome change

### PR DESCRIPTION
**Pull request by despotate**
Note: I know it says I changed 500 lines, but that's just from moving everything around relative to itself. None of it actually changes the code or gameplay. **please merge without looking so you can preserve your sanity**

- Reorders the effects to start at stage 1 and end at special
- Makes Babel use max_count instead of hackish vars and conditionals 
- Babel's activation message for when the user only knows one language now makes more sense
- ??? Helpful comments ???
- I swear I'm going to start actually adding features any day now instead of autistic code cleanups

### CODE REVIEW

File: code/modules/virus2/effect.dm [hunk 1]
```diff
@@ -834,305 +723,410 @@ datum/disease2/effect/lubefoot/deactivate(var/mob/living/carbon/mob)
...
+	max_count = 1
 
+	var/list/original_languages = list()
 
-/datum/disease2/effect/drained
-	name = "Drained Feeling"
-	stage = 1
+/datum/disease2/effect/babel/activate(var/mob/living/carbon/mob)
+	if(mob.languages.len <= 1)
+		to_chat(mob, "Your knowledge of language is just fine.")
```
Comment (gbasood) [inline code/modules/virus2/effect.dm:1066]: Ehhh
It makes more sense out of context, sure
Might want to find a new message instead

**Comment by gbasood** [comment]
Thank you for finding out more about virus code than I did to make babel happen
And also for cleaning up that horror show

File: code/modules/virus2/effect.dm (hunk 1 above)
Comment (despotate) [reply code/modules/virus2/effect.dm:1066]: I must have Babel syndrome because I don't understand what you're saying

Comment (gbasood) [reply code/modules/virus2/effect.dm:1066]: I meant that this message when displayed alone means absolutely nothing.
Imagine getting a one-line message saying "You're okay." out of nowhere.

**Pull request merged**